lib/cndb/omf.h

```diff
@@ -209,6 +219,26 @@ OMF_SETGET(struct cndb_kvset_del_omf, kvset_del_txid, 64);
 OMF_SETGET(struct cndb_kvset_del_omf, kvset_del_cnid, 64);
 OMF_SETGET(struct cndb_kvset_del_omf, kvset_del_kvsetid, 64);
 
+/**
+ * struct cndb_kvset_move_omf
+ *
+ * A KVSET_MOVE record persists the intent to move one or more kvsets from
+ * a source node to a target node
+ */
+struct cndb_kvset_move_omf {
+    struct cndb_hdr_omf hdr;
+    uint64_t            kvset_move_cnid;
+    uint64_t            kvset_move_src_nodeid;
+    uint64_t            kvset_move_tgt_nodeid;
+    uint32_t            kvset_move_pad;
+    uint32_t            kvset_move_kvset_idc;
+    uint64_t            kvset_move_kvset_idv[0];
+} HSE_PACKED;
```

Comment:
Change this is use a flexible array at the end (i.e., lose the zero in the brackets).